app/build.gradle

```diff
@@ -61,7 +61,8 @@ dependencies {
     implementation 'com.google.firebase:firebase-firestore-ktx'
     implementation 'com.google.firebase:firebase-auth:21.0.4'
     implementation 'com.firebaseui:firebase-ui-auth:8.0.1'
-    implementation 'com.mapbox.search:mapbox-search-android-ui:1.0.0-beta.29'
+    implementation 'com.mapbox.search:mapbox-search-android-ui:1.0.0-beta.30'
+    implementation 'com.google.android.gms:play-services-location:19.0.1'
```

Comment:
As you already change build.gradle, could you remove the warnings ?

Reply:
Good point, done